templates/prelogin/home.html

```diff
@@ -2,6 +2,33 @@
 {% load i18n %}
 {% load hq_shared_tags %}
 
+{% block js-inline %}
+<!-- Start of Async Drift Code -->
+<script>
+    !function() {
+        var t;
+        if (t = window.driftt = window.drift = window.driftt || [], !t.init) return t.invoked ? void (window.console && console.error && console.error("Drift snippet included twice.")) : (t.invoked = !0,
+        t.methods = [ "identify", "config", "track", "reset", "debug", "show", "ping", "page", "hide", "off", "on" ],
+        t.factory = function(e) {
+            return function() {
+                var n;
+                return n = Array.prototype.slice.call(arguments), n.unshift(e), t.push(n), t;
+            };
+        }, t.methods.forEach(function(e) {
+            t[e] = t.factory(e);
+        }), t.load = function(t) {
+            var e, n, o, i;
+            e = 3e5, i = Math.ceil(new Date() / e) * e, o = document.createElement("script"),
+                o.type = "text/javascript", o.async = !0, o.crossorigin = "anonymous", o.src = "https://js.driftt.com/include/" + i + "/" + t + ".js",
+                n = document.getElementsByTagName("script")[0], n.parentNode.insertBefore(o, n);
+        });
+    }();
+    drift.SNIPPET_VERSION = '0.3.1';
+    drift.load('7456r8nz2u34');
+</script>
+<!-- End of Async Drift Code -->
+{% endblock js-inline %}
+
```

Comment:
since this code is repeated in 3 places it might be nice to add it as its own template in the `partials` directory, then include it on each page. so we can make changes to each instance all at once. you can add the hubspot integration code to that partial template as well.